flax/nnx/rnglib.py

```diff
@@ -58,6 +58,7 @@ def _to_keyless(
 
 
 def _function_to_method(random_f):
```

Comment:
Necessary so that tabulate can identify calls to the rng random methods in its tables.